### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by rubencaro** [karan/atom-terminal#48]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (/usr/share/atom/resources/app.asar/src/package.js:810:9)
Package.hasActivationCommands (/usr/share/atom/resources/app.asar/src/package.js:735:20)
<unknown> (/usr/share/atom/resources/app.asar/src/package.js:189:24)
Package.measure (/usr/share/atom/resources/app.asar/src/package.js:167:15)
```

Created from atom 0.198.0

**Comment by tkw1536** [comment]
The exact same as #45. Please do not post double issues.

**Comment by andya9** [comment]
I fixed all deprecations in #41 (to be merged)
